clcache.py

```diff
@@ -192,7 +192,7 @@ def clean(self, stats, maximumSize):
         for o in objects:
             try:
                 objectInfos.append((os.stat(o), o))
-            except WindowsError:
+            except OSError:
```

Comment:
`OSError` is the only documented exception type for os.stat, so whatever the concrete type is, is implementation detail.

> Note: All functions in this module raise OSError in the case of invalid or inaccessible file names and paths, or other arguments that have the correct type, but are not accepted by the operating system.

https://docs.python.org/2/library/os.html
https://docs.python.org/2/library/os.html#os.stat